transports/quic/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "libp2p-quic"
+version = "0.8.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+description = "TLS based QUIC transport implementation for libp2p"
+repository = "https://github.com/libp2p/rust-libp2p"
+license = "MIT"
+
+[dependencies]
+if-watch = "1.1.1"
```

Comment:
Please, consider using 2.0.0. The upgrade procedure is not straitforward and libp2p shifts towards it.